### DISCUSSION TITLE
Bugs

### DISCUSSION THREAD

**Discussion by Yamidashka** [python-rope/rope#707] [Ideas]
The program is excellent, but there are a few drawbacks. I just wanted to let you know in case you take note of them. 1) The "Find Face" button doesn't work well. In the previous version, if I selected the face of Harry's father in the Spider-Man movie, then it wasn't possible to replace the face of the school bullies because the "Find Face" button considered them the same, even though they are very different. In the new version, there are moments when it even identifies Peter's face as the same, but when I replace the face, it only replaces the face of the first person I scanned with the "Find Face" button. So, I understand that the "Find Face" button and the swap feature detect faces differently (in my experience, the swap feature works much better). Because of this, I have to do multiple passes on the video to replace each of their faces. 2) The hair keeps flickering constantly, even if the face is static. Overall, the application is amazing, but these flaws are quite bothersome. I don't know whose fault it is, yours or the one who made the portable version, but you can't rewind the video during playback, otherwise, the program crashes.silence this warning we advise to pass rcond=None, to keep using the old, explicitly pass rcond=-1.
P = np.linalg.lstsq(X_homo, Y)[0].T # Affine matrix. 3 x

**Comment by lieryan** [comment]
Hi @Yamidashka, appreciate your taking the time to write a bug report but this is not the correct project to file this bug into. This repository is for a Python refactoring tool, not a face detection library.
